tests/puppeteer/campaigns/functional/BO/international/localization/languages/01_filterLanguages.js

```diff
@@ -0,0 +1,105 @@
+require('module-alias/register');
+// Using chai
+const {expect} = require('chai');
+const helper = require('@utils/helpers');
+const loginCommon = require('@commonTests/loginBO');
+// Importing pages
+const BOBasePage = require('@pages/BO/BObasePage');
+const LoginPage = require('@pages/BO/login');
+const DashboardPage = require('@pages/BO/dashboard');
+const LocalizationPage = require('@pages/BO/international/localization');
+const LanguagesPage = require('@pages/BO/international/languages');
+// Importing data
+const {Languages} = require('@data/demo/languages');
+
+let browser;
+let page;
+let numberOfLanguages = 0;
+
+// Init objects needed
+const init = async function () {
+  return {
+    boBasePage: new BOBasePage(page),
+    loginPage: new LoginPage(page),
+    dashboardPage: new DashboardPage(page),
+    localizationPage: new LocalizationPage(page),
+    languagesPage: new LanguagesPage(page),
+  };
+};
+
+describe('Filter Languages', async () => {
+  // before and after functions
+  before(async function () {
+    browser = await helper.createBrowser();
+    page = await helper.newTab(browser);
+    this.pageObjects = await init();
+  });
+  after(async () => {
+    await helper.closeBrowser(browser);
+  });
+
+  // Login into BO and go to contact page
+  loginCommon.loginBO();
+
+  it('should go to localization page', async function () {
+    await this.pageObjects.boBasePage.goToSubMenu(
+      this.pageObjects.boBasePage.internationalParentLink,
+      this.pageObjects.boBasePage.localizationLink,
+    );
+    await this.pageObjects.boBasePage.closeSfToolBar();
+    const pageTitle = await this.pageObjects.localizationPage.getPageTitle();
+    await expect(pageTitle).to.contains(this.pageObjects.localizationPage.pageTitle);
+  });
+
+  it('should go to languages page', async function () {
+    await this.pageObjects.localizationPage.goToSubTabLanguages();
+    const pageTitle = await this.pageObjects.languagesPage.getPageTitle();
+    await expect(pageTitle).to.contains(this.pageObjects.languagesPage.pageTitle);
+  });
+
+  it('should reset all filters and get number of languages in BO', async function () {
+    numberOfLanguages = await this.pageObjects.languagesPage.resetAndGetNumberOfLines();
+    await expect(numberOfLanguages).to.be.above(0);
+  });
+
+  // 1 : Filter languages with all inputs and selects in grid table
+  describe('Filter languages', async () => {
+    const tests = [
+      {args: {filterType: 'input', filterBy: 'id_lang', filterValue: Languages.english.id}},
+      {args: {filterType: 'input', filterBy: 'name', filterValue: Languages.english.name}},
+      {args: {filterType: 'input', filterBy: 'iso_code', filterValue: Languages.english.isoCode}},
+      {args: {filterType: 'input', filterBy: 'language_code', filterValue: Languages.english.languageCode}},
+      {args: {filterType: 'input', filterBy: 'date_format_lite', filterValue: Languages.english.dateFormat}},
```

Comment:
Does this mean you filter with a date value of `'m/d/Y'` It is a format not a date value, shouldn't it be tested with a real date?

Reply:
it's for Date format input filter 
![image](https://user-images.githubusercontent.com/48441421/71827156-eea5d480-309f-11ea-8665-241bb2351948.png)